docs/docker_deployment_guide.md

```diff
@@ -306,9 +306,16 @@ docker compose exec redis redis-cli
 
 ## Backup and restore
 
+Backup:
+
+```bash
+docker compose exec -it db pg_dump -U kbin kbin > dump.sql
+```
+
+Restore:
+
 ```bash
-docker exec -it container_id pg_dump -U kbin mbin > dump.sql
-docker compose exec -T database psql -U kbin mbin < dump.sql
+cat dump.sql | docker compose exec -i db psql -U kbin kbin
```

Comment:
hmm, this gets me `the input device is not a TTY`. I tried without `-i` as well

Reply:
ow maybe it should have been `-T`

Reply:
I trhink it will change it back.